modules/wallet/wallet_test.go

```diff
@@ -15,6 +16,16 @@ import (
 	"github.com/NebulousLabs/Sia/types"
 )
 
+var (
+	// Sometimes during testing, you get a statement
+	// `if err != expectedErr { return err }`. But if err == nil, then the
+	// calling function will think that the test has passed, even though the
+	// test failed. [This is a developer error, instead return
+	// errors.New("expecting..")] The new type of error catches when the
+	// developer has made a mistake and failed to catch a test suite failure.
+	errTestCompleted = errors.New("test complete")
```

Comment:
>  If you're expecting a specific error, you must write...

and if you space out and forget to do that, you might put the entire test suite at risk by silently failing to run half of a function. It's doubly bad because 

```go
if err != errBenignErr {
    return err
} 
```
is such a common pattern.

A silently failing test suite is tragic, especially because it's such an easy mistake to make. You get strong, reliable code when you write in ways that make it difficult make mistakes.

I'll switch it to panics. I feel strongly about this.